rakefile.rb

```diff
@@ -74,7 +75,7 @@
 
 desc "Executes xUnit tests using Mono"
 xunit :xunitmono => [] do |xunit|

```

Comment:
Hmm this looks like and old task? I don't think we have that on any other RAKE-files, if not then we should kill it I think?